scientist-softserv/harvard-curiosity
# Headings- override from out of the box spotlight

**Headings- override from out of the box spotlight**
* * *

**JIRA Ticket**: 
* https://github.com/harvard-lts/CURIOSity/issues/86

# What does this Pull Request do?
Uses CSS to make H1's look like H2's, H2's look like H3's, and H3's look like H4's, stylistically.

# How should this be tested?
Refer to [Harvard Library Brand Guidelines](https://library.harvard.edu/brand-guidelines#typography) to see what our typography and font styles looks like. Then you can check for the following:
* When inspected, page titles are coded as H1's, but look like H2's (a spot check should work here, as page-titles exist on all pages except the browse homepage, search results, and item detail pages; on the homepage, "show title" also needs to be turned on with a checkbox in order to see it)
* When inspected, section headings are coded as H2's, but look like H3's ([test page](https://localhost:21407/demo/feature/exhibit-item-widgets))
* When inspected, headings within a widget's text are coded as H3's, but look like H4's ([test page](https://localhost:21407/demo/feature/exhibit-item-widgets)) *note: if you don't see any of these, you may need edit the widgets on the page to include headings

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? NA
- integration tests? NA

# Interested parties
